src/main/java/hudson/plugins/git/UserRemoteConfig.java

```diff
@@ -75,19 +80,22 @@ public String toString() {
     public static class DescriptorImpl extends Descriptor<UserRemoteConfig> {
 
         public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Item project,
```

Comment:
The "new" best practice is to mark all methods that are expected only to be invoked by Jelly EL / Groovy EL as `@Restricted(NoExternalUse.class)` so that it is easier to change method signatures... your call as to whether you want to adopt that policy for the git plugin